### ISSUE TITLE
Mulitlingual support

### ISSUE THREAD

**Issue by slashme** [slashme/parliamentdiagram#29]
See https://commons.wikimedia.org/wiki/File:Multilingual_SVG_example.svg - this would be good for different scripts of numerals.

**Comment by BodhisattwaMandal** [comment]
Add {{translate|switch=yes}} while uploading.

**Comment by slashme** [comment]
Thanks, will do that soon - maybe this evening.

**Comment by slashme** [comment]
Oops, this of course isn't a fix for the issue: it's just a template to allow people to fix the issue. I actually need to add code to create numbers in multiple scripts. I'll leave the multilanguage template up on the beta version of the code for now - this will hopefully remind me to get this sorted.

**Comment by belett** [comment]
Yeah, I just remove the {{translate|switch=yes}} on a couple of file a there was indeed no <switch> in these SVG.
Could you sort this?
